Buildings/Resources/Scripts/Dymola/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/Validation/EfficiencyCondition.mos

```diff
@@ -0,0 +1,10 @@
+﻿simulateModel("Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.SetPoints.Subsequences.Validation.EfficiencyCondition", startTime=0,tolerance=1e-6, stopTime=7200, outputInterval=1, method="dassl", resultFile="EfficiencyCondition1");
+removePlots(false);
+createPlot(id=1, position={0, 0, 1660, 998}, y={"effCon.uQReq", "effCon.uQUpMin"}, range={0.0, 7200.0, 0.9, 1.6}, grid=true, colors={{28,108,200}, {238,46,47}});
+createPlot(id=1, position={0, 0, 1660, 329}, y={"effCon.uHotWatFloRat", "effCon.uUpMinFloSet"}, range={0.0, 7200.0, -0.2, 1.2000000000000002}, grid=true, subPlot=2, colors={{28,108,200}, {238,46,47}});
+createPlot(id=1, position={0, 0, 1660, 329}, y={"effCon.yEffCon"}, range={0.0, 7200.0, -0.2, 1.2000000000000002}, grid=true, subPlot=3, colors={{28,108,200}});
+ExportPlotAsImage("./Resources/Images/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/EfficiencyCondition1.png", 1, true, false);
+
+createPlot(id=2, position={0, 0, 1660, 998}, y={"effCon1.uQReq", "effCon1.uQDes"}, range={0.0, 7200.0, 0.78, 1.02}, grid=true, colors={{28,108,200}, {238,46,47}});
+createPlot(id=2, position={0, 0, 1660, 496}, y={"effCon1.yEffCon"}, range={0.0, 7200.0, -0.1, 1.1}, grid=true, subPlot=2, colors={{28,108,200}});
+ExportPlotAsImage("./Resources/Images/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/SetPoints/Subsequences/EfficiencyCondition2.png", 2, true, false);
```

Comment:
Why do we need this line? Does it do the export each time the script is run? I know we usually don't have it here. Did you just use it to create the plot images for info sections?

Reply:
Yes, that was the intention with which I added that line. Along with providing a dynamic link between the block and validation model, I thought having this line would ensure the latest validation plot was used in the information section whenever the model was updated. Will remove it if you think it can be potentially problematic in the future

Reply:
I checked with Michael. Even though this seems like a good idea, we would prefer not to have this line of code and just to have the developer updated the image as needed. The main reason is the path that does not need to be maintained by users of the model. There are also concerns related to binaries under different OSs. The validation results don't really need to be plotted in the info sections unless it makes a lot of sense.